### PR TITLE
feat: Add condition_report qualification in home theme

### PR DESCRIPTION
**Pull request by Merkur39**

### CODE REVIEW

**Review by paultranvan — COMMENTED** [review]
Are you sure about the "Inventory of accomodation" meaning? I feel like the correct expressions is more "Condition report"

File: packages/cozy-client/src/models/document/locales/en.json [hunk 1]
```diff
@@ -52,6 +52,7 @@
       "house_sale_agreeement": "House sale agreement |||| House sale agreements",
       "identity_photo": "Identity photo |||| Identity photos",
       "identity": "Identity",
+      "inventory_of_accommodation": "inventory of accommodation - entry/exit",
```
Comment (paultranvan) [inline packages/cozy-client/src/models/document/locales/en.json:55]: Missing uppercase on "inventory"

**Comment by Merkur39** [comment]
> Are you sure about the "Inventory of accomodation" meaning? I feel like the correct expressions is more "Condition report"

No preference, @BenjaminMty an opinion ?

**Comment by BenjaminMty** [comment]
> > Are you sure about the "Inventory of accomodation" meaning? I feel like the correct expressions is more "Condition report"
> 
> No preference, @BenjaminMty an opinion ?

ok pour "Condition report"

**Review by paultranvan — APPROVED** [review]

**Pull request merged**